test/core/consensus/vrf/vrf_threshold_test.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+extern "C" {
+#include <sr25519/sr25519.h>
+}
+
+#include <gtest/gtest.h>
+#include <array>
+#include <boost/multiprecision/cpp_int.hpp>
+
+using boost::multiprecision::uint256_t;
+
+uint256_t bytes_to_uint256_t(uint8_t const *output, size_t len) {
+  if (len % sizeof(boost::multiprecision::limb_type))
+    throw std::runtime_error("len % sizeof(mp::limb_type)");
```

Comment:
```suggestion
    FAIL() << "len % sizeof(mp::limb_type)";
```
Would be cleaner. Also, strange message, I think